manage_breast_screening/notifications/management/commands/message_batch_functions.py

```diff
@@ -0,0 +1,23 @@
+from datetime import datetime
+from zoneinfo import ZoneInfo
+
+from manage_breast_screening.notifications.models import (
+    Message,
+    MessageBatch,
+)
+
+TZ_INFO = ZoneInfo("Europe/London")
+
+
+def mark_batch_as_sent(message_batch: MessageBatch, response_json: dict):
```

Comment:
Although we can write Python in functional and OOP style I'm not sure we should.
I really like the functional style personally but Django admin commands are OOP in nature.
Perhaps a `CommandHelpers` or `MessageBatchHelpers` class with static or class methods we can share across the set of commands?

Reply:
that makes sense!